nixops/diff.py

```diff
@@ -30,7 +30,9 @@ def _default_handle(self):
         """
         raise NotImplementedError
 
-    def get_deps(self) -> List[Handler]:
+    # This is List[Handler], but that name doesn't exist inside the
+    # class definition.
+    def get_deps(self) -> List[Any]:
```

Comment:
I think this can be `List[Handler]` with `from __future__ import annotations`, give that a try?